project/project.ts

```diff
@@ -390,7 +392,7 @@ const shouldIncludePath = (path: string, stats: fs.Stats): boolean => {
     return false
   }
 
-  if (stats.isDirectory()) {
+  if (stats && stats.isDirectory()) {
```

Comment:
I think `stats?.isDirectory()` should do the same and be the more modern style. But doesn't really matter.

Reply:
Yeah, I'm using TypeScript for long time (form versions 2.x) and I have still habits of using old style code. Mainly because I work on codespaces were we still have old TS versions and not everything is supported 😛